decentraland/components/presentational/Parcel.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Image, StyleSheet, Text } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveHeight,
+} from "react-native-responsive-dimensions";
+
+export default function Parcel({ parcel }) {
+  const { img, name } = parcel;
+  return (
+    <React.Fragment>
+      <Image style={styles.landImage} source={{ uri: img }} />
+      <Text>{name}</Text>
+    </React.Fragment>
+  );
+}
+
+Parcel.propTypes = {
+  parcel: PropTypes.object.isRequired,
+};
+
+const styles = StyleSheet.create({
+  landImage: {
+    width: responsiveWidth(60),
```

Comment:
I think it might look better if the image filled up most of the screen's width.

Reply:
```suggestion
    width: responsiveWidth(95),
```

Reply:
We'd likely need to change the height too if we go with this. We could make height determined programmatically based on the width constant before this line if we want.

Reply:
Done on PR https://github.com/tasitlabs/tasit/pull/161

Reply:
Thanks! Resolving